src/s1/s1/static/js/Packages/S1/Workbench/Workbench.ts

```diff
@@ -294,7 +294,7 @@ var getMeeting = (proposal : RIProposal, process : RIS1Process) => {
         var proposalDecisionDate = AdhUtil.deepPluck(
             AdhProcess.getStateData(proposal.data[SIWorkflowAssignment.nick], proposalState), ["start_date"]);
 
-        return (processDecisionDate === proposalDecisionDate) ? "current" : "archive";
+        return (proposalDecisionDate <= processDecisionDate) ? "archive" : "current";
```

Comment:
> Am 24.08.2016 um 10:51 schrieb Tobias Bengfort <notifications@github.com>:
> 
> In src/s1/s1/static/js/Packages/S1/Workbench/Workbench.ts <https://github.com/liqd/adhocracy3/pull/2617#discussion_r76017832>:
> 
> > @@ -294,7 +294,7 @@ var getMeeting = (proposal : RIProposal, process : RIS1Process) => {
> >          var proposalDecisionDate = AdhUtil.deepPluck(
> >              AdhProcess.getStateData(proposal.data[SIWorkflowAssignment.nick], proposalState), ["start_date"]);
> >  
> > -        return (processDecisionDate === proposalDecisionDate) ? "current" : "archive";
> > +        return (proposalDecisionDate <= processDecisionDate) ? "archive" : "current";
> Just so we do not forget it. IIRC, these values are strings, so comparison might not work as expected
> 
Actually on rfc formatted date strings it does. Thats precisely the reason they are formatted the way they are. Or am I missing something?